CollapseLauncher/Classes/Helper/WindowUtility.cs

```diff
@@ -885,6 +885,16 @@ public static int SetProgressValue(nint hwnd, ulong completed, ulong total)
         {
             return Taskbar.SetProgressValue(hwnd, completed, total);
         }
+
+        public static int SetTaskBarState(TaskbarState state)
+        {
+            return Taskbar.SetProgressState(CurrentWindowPtr, state);
+        }
+
+        public static int SetProgressValue(ulong completed, ulong total)
+        {
+            return Taskbar.SetProgressValue(CurrentWindowPtr, completed, total);
```

Comment:
also in here

Reply:
Resolved in https://github.com/CollapseLauncher/Hi3Helper.Win32/commit/b62180e69319c29b5dba34c8e57f8692087c6429